app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,327 @@
+{{site.kic_product_name}} translates Kubernetes objects it gets from the Kubernetes API and pushes
+the translation result via Kong’s Admin API to {{site.base_gateway}} instances. However, issues
+can arise at various stages of this process:
+
+1. Admission Webhook: Validates individual Kubernetes objects against schemas and basic rules.
+2. Translation Process: Detects issues like cross-object validation errors.
+3. Kong Response: Kong rejects the configuration and returns an error associated with a specific object.
+
+Fallback Configuration is triggered when an issue is detected in the 3rd stage and provides the following benefits:
+- Allows unaffected objects to be updated even when there are configuration errors.
+- Automatically builds a fallback configuration that Kong will accept without requiring user intervention by
+  either:
+  - Excluding the broken objects along with its dependants.
+  - Backfilling the broken object along with its dependants using the last valid Kubernetes objects' in-memory cache (if `--use-last-valid-config-for-fallback` flag is set).
+- Enables users to inspect and identify what objects were excluded from or backfilled in the configuration using
+  diagnostic endpoints.
+
+Below table summarizes the behavior of the Fallback Configuration feature based on the configuration:
+
+| `FallbackConfiguration` feature gate value | `--use-last-valid-config-for-fallback` flag value | Behavior                                                                                                                                             |
+|--------------------------------------------|---------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `false`                                    | `false`/`true` (has no effect)                    | The last valid configuration is used as a whole to recover (if stored).                                                                              |
+| `true`                                     | `false`                                           | The Fallback Configuration is triggered - broken objects and their dependants are excluded.                                                          |
+| `true`                                     | `true`                                            | The Fallback Configuration is triggered - broken objects and their dependants are excluded and backfilled with their last valid version (if stored). |
+
+Below diagram illustrates how the Fallback Configuration feature works in detail:
+
+![fallback-config](/assets/images/products/kubernetes-ingress-controller/fallback-config.png "Fallback Configuration diagram")
+
+## Example Scenario
+
+In this example we'll demonstrate how the Fallback Configuration works in practice.
+
+{% include /md/kic/prerequisites.md release=page.release disable_gateway_api=false %}
```

Comment:
This block seems to be expandable on click with a bit of a surprising behavior:

Freshly opened page:

<img width="1304" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/739996/678c3edf-38ea-4a85-8a06-576d7858202a">

After clicking on the block:

<img width="1294" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/739996/38101134-1296-4ea7-988a-b6e1890c3791">

\+ long explanation how to install Gateway API and Kong via helm chart. There is no clear indication where the expanded block ends and when the instructions from this page continue.

Reply:
This also causes the TOC to be slightly misleading:

<img width="239" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/739996/296b2dbb-d91f-42b7-b50f-15b4c043cb9f">

Reply:
It is a reusable building block to be used with `include` directive. You're right it doesn't look good, especially the TOC. I moved it to the beginning of the guide (similarly as it is done in other [guides](https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/services/http/)) so it doesn't look weird.

Reply:
The styling is a pre-existing issue which is in the backlog